docs/mod-lists/general-mod-list.md

```diff
@@ -54,6 +54,18 @@ Select your Minecraft version below to view compatible mods.
     
     - Download: https://modrinth.com/mod/animations
     - Support: https://polyfrost.org/discord
+
+    ### OverflowAnimationsV2
+    An updated and mostly rewritten version of Sk1er's Old Animations mod. Has complete 1:1 parity with 1.7 animations, fixes most bugs and compatibility issues (including block-hit banning users from MineMenClub), and includes the custom animations feature from Dulkir and more modern version animations.
```

Comment:
Compared to descriptions of the other mods in this list, this one feels way too long. Any chance you can shorten/compact it more to feel more consistent with the rest?

Also I feel it would be more fair to keep the fork below the original in this list. This is just my personal opinion though..

Reply:
I second on that ^^^